services/sidecar/src/simcore_service_sidecar/executor.py

```diff
@@ -207,7 +206,6 @@ async def _write_input_file(self, inputs: Dict) -> None:
             file_name.write_text(json.dumps(inputs))
             log.debug("Writing input file DONE")
 
-    @retry(reraise=True, stop=stop_after_attempt(3))
```

Comment:
what happens if there is an error raised by the Docker API during pull (eg: the registry times out)?

Reply:
the error will be raised, celery will catch it and re-schedule the task

Reply:
OK, thought it was something like that, just wanted to make sure.